content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md

```diff
@@ -0,0 +1,137 @@
+---
+title: influxdb.api() function
+description: >
+  The `influxdb.api()` function submits an HTTP request to the specified InfluxDB
+  API path and returns a record containing the HTTP status code, response headers,
+  and response body as a byte array.
+menu:
+  influxdb_2_0_ref:
+    name: influxdb.api
+    parent: InfluxDB-exp
+weight: 401
+influxdb/v2.0/tags: [functions, influxdb, api]
+---
+
+The `influxdb.api()` function submits an HTTP request to the specified InfluxDB
+API path and returns a record containing the HTTP status code, response headers,
+and response body as a byte array.
+
+```js
+import "experimental/influxdb"
+
+influxdb.api(
+  method: "get",
+  path: "/example",
+  host: "http://localhost:8086",
+  token: "mySupeR53cre7t0k3n",
+  headers: ["header1": "header1Value", "header2", "header2Value"],
+  query: ["ex1": "example1", "ex2": "example2"],
+  timeout: 30s,
+  body: bytes(v: "Example body")
+)
+```
+
+#### Response record schema
```

Comment:
Does it make sense to keep in parallel verbal phrase with the usage.from() function here, like "View response data/record schema" even though this is shorter?

Reply:
Sure, I can do that.